metacatalog/ext/standard_export/util.py

```diff
@@ -0,0 +1,627 @@
+from typing import Union
+import os
+from datetime import datetime
+
+
+from jinja2 import Environment, FileSystemLoader
+import pandas as pd
+from lxml import etree
+import shapely
+from geoalchemy2.shape import to_shape
+from geoalchemy2.elements import WKBElement
+
+
+from metacatalog.models import Entry
+from metacatalog.util.results import ImmutableResultSet
+
+
+def _init_iso19115_jinja():
+        """
+        Initialize jinja environment for ISO 19115 export.
+
+        """
+        # folder containing ISO 19115 templates
+        absolute_path = os.path.dirname(__file__)
+        relative_path = "schemas/iso19115"
+        full_path = os.path.join(absolute_path, relative_path)
+        
+        # autoescape=False: fixes problems with symbol ">" (keyword.path)
+        env = Environment(loader=FileSystemLoader(searchpath=full_path), autoescape=False)
+        
+        # prevent whitespaces / newlines from jinja blocks in template
+        env.trim_blocks = True
+        env.lstrip_blocks = True
+
+        # get template
+        template = env.get_template("iso19115-2.j2")
+        
+        return template
+
+
+def _get_uuid(rs: ImmutableResultSet) -> str:
+    """
+    Returns uuid of ImmutableResultSet.  
+    Returns the uuid of the base group (if exists) or the concatenated uuids of all 
+    members.
+
+    Returns
+    ----------
+    uuid : str
+        Used for field <gmd:fileIdentifier> and field <gmd:identifier>, not repeatable.
+
+    """
+    # if a base group exists, use the uuid of the base group
+    if rs.group:
+        uuid = rs.group.uuid
+
+    # if no group exists, there is only one member in the ImmutableResultSet -> use its uuid
+    else:
+        uuid = rs.get('uuid')
+
+    return uuid
```

Comment:
Is this still necessary?
The `ImmutableResultSet.uuid` property should already return this, right?

Reply:
That's right!